packages/env/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Master
 
+## 1.1.0 (2020-02-12)
```

Comment:
```suggestion
```

Reply:
Just leave the `### New Feature` heading underneath `## Master`. When packages are next published, the version and date are added 🙂

https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs

Reply:
Ah I see. Thanks!